test/replicate-and-load.test.js

```diff
@@ -22,7 +22,7 @@ const ipfsPath2 = './orbitdb/tests/replicate-and-load/2/ipfs'
 
 Object.keys(testAPIs).forEach(API => {
   describe(`orbit-db - Replicate and Load (${API})`, function() {
-    this.timeout(config.timeout)
+    this.timeout(config.timeout * 2)
```

Comment:
What factors are responsible for the `* 2`?

Reply:
I believe CI was taking too long for the browser tests and it timed out.